### ISSUE TITLE
Copy button on installation instructions is not accessible

### ISSUE THREAD

**Issue by felixfbecker** [sourcegraph/sourcegraph#13438]
https://docs.sourcegraph.com/

![image](https://user-images.githubusercontent.com/10532611/91558225-3aaacf80-e936-11ea-9bef-7bdf4a753ffd.png)

![image](https://user-images.githubusercontent.com/10532611/91558199-2f57a400-e936-11ea-96e9-796d073531f0.png)

The button is not using a `<button>` element (is therefor not tab-focusable) and sets itself intentionally to `aria-hidden="true"`, making it invisible to screen readers. The best fix is to use a `<button>` element.

**Comment by umpox** [comment]
We don't actually have a copy icon anymore. The functionality to copy text and the icon library we were using have been [removed already](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/commit/ab4ddde8c96923df0e4e83b56578931ce1bc399f?visible=30).

I'm struggling to find more context on if this, but if we feel we should re-add this functionality I'd propose closing this ticket and I'll create a new one to add copy functionality to _all_ code blocks in our docs page
